ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -288,11 +287,6 @@ public String findPatientId() {
         return findElement(PATIENT_ID).getText();
     }
 
-    public AllergyPage clickOnAllergyManagement() {
```

Comment:
For software quality and readability, Have replaced the implementation of clickOnAllergyManagement()  with clickOnAllergiesWidgetLink() since they all perform the same goal. Updated the respective child classes within the module that implement the functionality of the page. However, am skeptical about other modules elsewhere that use this functionality through a library But for the sake of software quality its a good option to make some refactoring in our base module